src/app/loadout/loadout.service.ts

```diff
@@ -14,7 +14,7 @@ import { default as reduxStore } from '../store/store';
 import * as actions from './actions';
 import { loadoutsSelector } from './reducer';
 
-export const enum LoadoutClass {
+export enum LoadoutClass {
```

Comment:
are enums implied const?

Reply:
No, they overloaded const a bit here. A const enum will just get replaced with its value everywhere, while a non-const enum generates a table you can use to look up the enum names and stuff.